announcements/2021/week17.md

```diff
@@ -0,0 +1,20 @@
+# :zap: This week in provider-independent announcements (April 30, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Cert Manager v2.6.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#260---2021-04-30) adds support for DNS01 challenge, allowing gigantic.io certificates to work behind a firewall. It simplifies creation of installations on-premises. Prior, [Cert Manager v2.5.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#250---2021-04-19) upgrades to upstream v1.2.0.
+
+[NGINX IC v1.16.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1161---2021-04-20) adds pass through annotation and labels to the controller service...?
```

Comment:
@webwurst , can you help me - how can a user benefit from this change? :) Or, why did we make this change?

Reply:
Sorry, don't remember the details. It's just more flexible and straight from upstream.